modulo2/projeto-labefy/projeto-labefy/src/App.js

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import axios from "axios"
+import styled components from "styled-components"
+
+
+
+
+export default  class App extends React.Component{ 
+    const executeRequest =  async () => {
+        const body = {
+            name = ""
+        };
+        
+        const response = await axios.post("https://us-central1-labenu-apis.cloudfunctions.net/labefy/playlists",
+        body,
+        {
+        headers: {
+            Authorization: "fernando-taveiraorsini-carver"   
+        }
+
+        }
+             
+        );
+
+        console.log(response)
+         
+    } 
+      
+    render() {
+      return (
+        <div>
+          <h2>Playlist</h2>
+          <button onClick={executeRequest}>Criar Playlist</button>
+        </div>
```

Comment:
Quando uma função que está localizada antes do render é chamada  no JSX ela deve ter o this. antes.
ex:

`this.executeRequest`